x/marker/keeper/marker.go

```diff
@@ -701,20 +701,19 @@ func (k Keeper) RemoveSendEnabledForMissingMarkers(ctx sdk.Context) {
 	ctx.Logger().Info("Removing SendEnabled for Missing Markers...")
```

Comment:
Not sure this function is safe ... it seemingly only works by convention --- that there will be (or was) a marker with every send enabled entry ... but perhaps there could be one at some point from say an IBC denom...

Reply:
I didn't look into usage of this function. I guessed that it was from a migration. I'll look into removing it.

Reply:
Yeah. It's only called as part of the `lava` migration. I'll create an issue to clean up the upgrades and remove upgrade-only code.

Reply:
Hmm. I wish we had put upgrade in that name then... but yeah, good idea to remove now so it doesn't get used on accident

Reply:
Done.